UNFLoader/Makefile

```diff
@@ -1,4 +1,5 @@
 APP=UNFLoader
+OS_NAME := $(shell uname -s)
```

Comment:
`uname` is POSIX-compliant, so most non-Win32 OSes should handle this properly. We use this here to check if we're on macOS by seeing if it evaluates to `Darwin`.